app/index.html

```diff
@@ -49,11 +49,14 @@
       </div>
     </nav>
 
-    <div ng-view style="margin-top: 5%;"></div>
+    <div class="container">
+      <div ng-view style="margin-top: 2%;"></div>
+    </div>
 
     <!-- Optional JavaScript -->
     <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="http://code.jquery.com/jquery-1.10.2.js"></script>
+    <!-- <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrit1y="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script> -->
```

Comment:
You can just remove commented out code like this to keep the codebase cleaner

Reply:
I removed the unneeded comments, created a lib folder and downloaded all the used libraries locally.

Reply:
Wow this looks better. Do you know from your experience, is it more common to do it the first way or to download the libraries locally like this? I can see pros and cons to both.